### PR TITLE
Move SQL utilities from bgbb_lib

### PR DESCRIPTION
**Pull request by acmiyaguchi**
This moved most of the application-like code from bgbb_lib into this repo, so it can be migrated to a GCP workflow.

### CODE REVIEW

File: bgbb_airflow/fit_airflow_job.py [hunk 1]
```diff
@@ -2,17 +2,12 @@
...
-from bgbb.sql.sql_utils import (
-    S3_DAY_FMT_DASH,
-    reduce_rec_freq_spk,
-    run_rec_freq_spk,
-)
 from pyspark.sql import SparkSession
 
+from bgbb import BGBB
 from bgbb_airflow.bgbb_utils import PythonLiteralOption
-import bgbb_airflow
```
Comment (wcbeard) [inline bgbb_airflow/fit_airflow_job.py:14]: Won't this break the ` bgbb_airflow.__version__` in line 114 below?

**Review by wcbeard — CHANGES_REQUESTED** [review]
All good, except for a question on `fit_airflow_job.py`

File: bgbb_airflow/fit_airflow_job.py (hunk 1 above)
Comment (acmiyaguchi) [reply bgbb_airflow/fit_airflow_job.py:14]: Yes, this will. I'll run the jobs manually to double check that this PR won't break the current workflow.

**Comment by acmiyaguchi** [comment]
```
./bin/run-databricks \
    --token ${DB_TOKEN} \
    --git-path https://github.com/acmiyaguchi/bgbb_airflow.git \
    --git-branch sql-utils \
    bgbb_fit \
        --submission-date 2019-11-01 \
        --prefix bgbb/params/v1 \
        --bucket telemetry-test-bucket
```
https://dbc-caf9527b-e073.cloud.databricks.com/#job/5184/run/1

```
./bin/run-databricks \
    --token ${DB_TOKEN} \
    --git-path https://github.com/acmiyaguchi/bgbb_airflow.git \
    --git-branch sql-utils \
    bgbb_pred \
        --submission-date 2019-11-01 \
        --param-bucket telemetry-parquet \
        --param-prefix bgbb/params/v1 \
        --pred-bucket telemetry-test-bucket \
        --pred-prefix bgbb/active_profiles/v1
```
~https://dbc-caf9527b-e073.cloud.databricks.com/#job/5183/run/1~
https://dbc-caf9527b-e073.cloud.databricks.com/#job/5185/run/1

**Comment by acmiyaguchi** [comment]
Both of the jobs successfully completed, this is ready for another look.

**Review by wcbeard — APPROVED** [review]

**Pull request merged**